resque-multi-job-forks.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.description = "When your resque jobs are frequent and fast, the overhead of forking and running your after_fork might get too big."
 
   # Depends on minor version, due to monkeypatches Resque::Worker internals.
-  s.add_runtime_dependency("resque", ">= 1.27.0", "< 2.1")
+  s.add_runtime_dependency("resque", ">= 1.27.0", "< 2.2")
```

Comment:
👋 Thanks for getting this work started! We also ran into it recently. However, the issue that we encountered was that when Resque updated to 2.1.0, our dependency on resque-multi-job-forks was automatically pulled _down_ to v0.1.1 (released in 2010). We were surprised by such an old release being included in our build.

This change certainly fixes the issue for Resque 2.1.x, but I'm concerned about a similar situation happening when Resque 2.2.0 is released in the future. Would it be possible to consider removing the upper limit on this dependency? Or perhaps pinning it to the major version 3.x instead of minor versions of 2.x.y? Another way of asking is, are there known issues with future minor versions of Resque or are we trying to safe-guard? If it's the latter we're concerned with, we might consider that bundle automatically resolving such an old version is even more risky than allowing future version of Resque to go unchecked by this lib.

Thoughts?

Reply:
We ran into it as well. It might be a good idea to yank 0.1.0 and 0.1.1 from Rubygems, if the maintainer still has access to do that. They're the only ones without maximum versions specified.

Reply:
> might be a good idea to yank 0.1.0 and 0.1.1 from Rubygems

Great idea! 🙏

Reply:
Thanks guys. 

0.1.0 and 0.1.1 have been yanked.